main.tf

```
@@ -0,0 +1,119 @@
+locals {
+  app_name_and_env = "${var.app_name}-${data.terraform_remote_state.common.outputs.app_env}"
+  app_env          = data.terraform_remote_state.common.outputs.app_env
+  app_environment  = data.terraform_remote_state.common.outputs.app_environment
+  name_tag_suffix  = "${var.app_name}-${var.customer}-${local.app_environment}"
+}
+
+/*
+ * Create cloudwatch log group for app logs
+ */
+resource "aws_cloudwatch_log_group" "cw_b2_fsbackup" {
+  name              = "${var.app_name}-${var.app_env}"
+  retention_in_days = 14
+
+  tags = {
+    name = "cloudwatch_log_group-${local.name_tag_suffix}"
+  }
+}
+
+/*
+ * Create required passwords
+ */
+resource "random_id" "b2_repo_password" {
+  byte_length = 16
+}
+
+/*
+ * Create task definition for file system backup
+ */
+locals {
+  task_def = templatefile("${path.module}/task-definition-b2-backup.tftpl",
+    {
+      atlas_token           = var.tfc_access_token
+      b2_access_key         = data.terraform_remote_state.b2-bucket.outputs.application_key_id
+      b2_secret_key         = data.terraform_remote_state.b2-bucket.outputs.application_key_secret
+      fsbackup_mode         = var.b2_fsbackup_mode
+      tfc_organization_name = var.tfc_organization_name
+      backup_args           = var.b2_fsbackup_args
+      forget_args           = var.b2_fsbackup_forget_args
+      restic_host           = var.b2_fsbackup_host
+      repo_pw               = random_id.b2_repo_password.hex
+      repo_string           = var.b2_fsbackup_repo
+      restic_tag            = "${var.app_name}-${var.app_env}-b2-fs-backup"
+      source_path           = var.backup_path
+      cpu                   = var.cpu
+      memory                = var.memory
+      docker_image          = "silintl/tfc-backup-b2"
+      docker_tag            = var.docker_tag
+      #  app_env               = var.app_env
```

Depending on why this is here:
```suggestion
      #  app_env               = local.app_env
```